### DISCUSSION TITLE
【2022.12.28】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1938] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by starComingup** [comment]
Distinguish between app and cluster layer images, cri (docker,containerd), runtime(k0s.k3s,k8s) info. Could implement by image annotations? How to set these annotations?

**Reply by starComingup** [reply]
Do not need distinguish layer images. it implemented by build stage.
action: begin to define cri and runtime in image spec.

**Comment by jsparter** [comment]
brief introduction of Skopeo

**Comment by kakaZhou719** [comment]
1. 期望能够在image spec中，存储runtime信息，比如cri runtime信息，cluster runtime信息@周欣元，进而能够在build阶段实现，需要和@裕惺讨论。
2. Skopeo介绍分享，目前skopeo copy存储若支持 registry后端存储，仍需要在build阶段启动registry service。考虑能否在run阶段 对镜像格式做处理，后续会提交issue讨论sealer对容器镜像的存储格式。@吴菲祺
3. E2E自测完成，代码已经提交待检视，希望0.9.1合入。@左海余
